README.md

```diff
@@ -1,64 +1,97 @@
+# An OpenWrt UBI Installer Image Generator for Linksys E8450 and Belkin RT3200
 
 ![animated gif showing web UI and serial during installation](https://user-images.githubusercontent.com/9948313/108781223-78915500-7561-11eb-851a-3c4c744ad6c2.gif)
-(The serial interface is displaying RX only for documentation purposes as the only interaction required is from within the web browser via HTTP file upload.)
 
-This script downloads the OpenWrt ImageBuilder to generate a release-like (ie. including LuCI) sysupgrade image. 
-The process involves re-packaging the initramfs image to contain everything necessary for a permanent recovery image within NAND flash including the installer script and the prerequisite installation images.
+*Animated GIF showing the installation process. The serial interface on the right is displaying RX for documentation purpose only. The interaction required is shown on the left, which is done entirely within the web browser.*
 
-The resulting file `openwrt-mediatek-mt7622-linksys_e8450-ubi-initramfs-recovery-installer.itb` is suitable to be flashed by the vendor firmware Web-UI as well as non-UBI OpenWrt running on the device (use `sysupgrade -F openwrt-mediatek-mt7622-linksys_e8450-ubi-initramfs-recovery-installer.itb`).
+This script downloads the OpenWrt ImageBuilder to generate a release-like (ie. LuCI included) *sysupgrade* image. The process involves re-packaging the *initramfs* image to contain everything necessary for a permanent recovery image within the NAND flash, including the installer script and the pre-requisite installation images.
 
-**WARNING** This will replace the bootloader (TF-A 2.4, U-Boot 2021.10) and convert the flash layout of the device to UBI! The installer stores a copy of the previous bootchain in a dedicated UBI volume `boot_backup`. If you want to go back to the vendor firmware, you will have to boot into recovery mode (ie. initramfs),
-copy those files into `/tmp`, umount the UBI volume, detach the UBI device and then write the files to the corresponding MTD partitions (mtd write $file /dev/mtdX).
+The resulting file `openwrt-mediatek-mt7622-linksys_e8450-ubi-initramfs-recovery-installer.itb` can be flashed via the vendor/official firmware web interface, as well as OpenWrt firmware running non-UBI builds (by running `sysupgrade -F openwrt-mediatek-mt7622-linksys_e8450-ubi-initramfs-recovery-installer.itb`).
 
-To be on the safe side, it is still recommended to make a complete backup of the device flash __**before**__ running the installer!
+**WARNING #1** This will replace the bootloader (TF-A 2.4, U-Boot 2021.10) and convert the flash layout of the device to UBI! The installer stores a copy of the previous bootchain in a dedicated UBI volume `boot_backup`.
 
-To use the script to generate the installer image, you will need all runtime requirements of the OpenWrt ImageBuilder installed as well as `libfdt-dev` and `cmake` which is needed by the installer-generator itself.
```

Comment:
You are removing the requirement for the **run-time dependencies** of the ImageBuilder here...